spec/models/spree/line_item_spec.rb

```diff
@@ -197,6 +197,14 @@ module Spree
         end
 
         it "includes soft-deleted products/variants" do
+          # Various erros on destroying a product, trying to destroy stock items for master,
+          # then finding they don't exist. Maybe the callbacks were doing some extra juju before...
+
+          pp v1
+          pp v1.stock_items
+          pp li3.quantity
+
```

Comment:
Thank you for the explanation. Can we delete the debug output now? Maybe move the comment into the commit message and just explain what the reload is doing?
```suggestion
```

Reply:
Oooooops, that wasn't meant to be commited :sweat_smile:  This was from a point when I was debugging issues with the variant factory setup that was causing lots of weird failures, but it's fixed now. :+1: